src/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Net.Http
     [SuppressMessage("Microsoft.Serialization", "CA2229")]
     public class HttpRequestException : Exception
     {
-        internal bool AllowRetry { get; }
+        internal RequestRetryType AllowRetry { get; }
```

Comment:
This is now relying on RequestRetryType.NoRetry having the value 0.  That feels a little brittle to me.  Even though it's the default, consider adding ` = RequestRetryType.NoRetry;` to this to make it explicit.  If the values were to change in the future, it'd lead to very subtle and hard to find behavioral impact.